plugins/Goals/lang/en.json

```diff
@@ -3,63 +3,63 @@
+        "ColumnOrdersDocumentation": "The total number of e-commerce orders which contained this %s at least once.",
+        "ColumnPurchasedProductsDocumentation": "The number of purchased products is the sum product quantity of all e-commerce orders sold.",
         "ColumnQuantityDocumentation": "Quantity is the total number of products sold for each %s.",
         "ColumnRevenueDocumentation": "The total revenue generated by %s.",
         "ColumnRevenuePerVisitDocumentation": "The total revenue generated by %s divided by the number of visits.",
-        "ColumnRevenuePerEntryDocumentation": "The total revenue generated by %s divided by the number of entrances.",
+        "ColumnRevenuePerEntryDocumentation": "The total revenue generated by %s divided by the number of customer entries.",
         "ColumnVisits": "The total number of visits, regardless of whether a goal was triggered or not.",
-        "ColumnVisitsProductDocumentation": "The number of visits on the Product\/Category page. This is also used to process the %s conversion rate. This metric is in the report if Ecommerce view tracking was setup on Product\/Category pages.",
+        "ColumnVisitsProductDocumentation": "The number of visits on the Product\/Category page. Also used to process the %s conversion rate. This metric is in the report if e-commerce view-tracking was set up on the Product\/Category pages.",
         "Contains": "contains %s",
-        "ConversionByTypeReportDocumentation": "This report provides detailed information about the goal performance (conversions, conversion rates and revenue per visit) for each of the categories available in the left panel. %1$s Please click on one of the categories to view the report. %2$s For more information, read the %3$sTracking Goals documentation%4$s",
+        "ConversionByTypeReportDocumentation": "This report provides detailed info about the goal performance (conversions, conversion rates and revenue per visit) for each of the categories available in the left panel. %1$s Please click on one of the categories to view the report. %2$s For more info, read the %3$sTracking goals documentation%4$s",
         "ConversionRate": "%s conversion rate",
         "ConversionRatePageViewedBefore": "Viewed before %s rate",
         "Conversions": "%s conversions",
         "ConversionsDescription": "conversions",
-        "ConversionsOverview": "Conversions Overview",
-        "ConversionsOverviewBy": "Conversions overview by type of visit",
+        "ConversionsOverview": "Conversion Overview",
+        "ConversionsOverviewBy": "Conversion overview by type of visit",
         "DaysToConv": "Days to Conversion",
-        "DaysToConvReportDocumentation": "This report shows how many days pass before your visitors convert a goal.",
+        "DaysToConvReportDocumentation": "This report shows how many days pass on average before your visitors convert a goal.",
```

Comment:
The report actually does not show the average.
![image](https://user-images.githubusercontent.com/1579355/173031375-757c51bd-90d4-4dc9-93b9-da79f9e92282.png)

```suggestion
        "DaysToConvReportDocumentation": "This report shows how many days pass before your visitors convert a goal.",
```